test/blackbox/helpers_scrub.bash

```diff
@@ -0,0 +1,99 @@
+ROOT_DIR=$(git rev-parse --show-toplevel)
```

Comment:
Why do we need separate helper files?

Reply:
Tests have different prerequisites/ helper functions . Even if we would use a general helpers.bash file, there would still be the need to import it in every helpers_*extension*.bash and wrap it with specific logic for that test.